### PR TITLE
fix arrow keys and local storage

### PR DESCRIPTION
**Pull request by WardCunningham**
I don't know what I was thinking when I replaced a convoluted switch statement with a malformed table lookup.
This expression has been shown to work.

### CODE REVIEW

**Review by paul90 — APPROVED** [review]

**Comment by paul90** [comment]
Could have just reverted to the original, from the decaffeinate script:
``` javascript
    const direction = (() => { switch (event.which) {
      case LEFTARROW: return -1;
      case RIGHTARROW: return +1;
    } })();
```

File: lib/legacy.js [hunk 1]
```diff
@@ -51,10 +51,9 @@ $(function() {
 
   const LEFTARROW = 37;
   const RIGHTARROW = 39;
-  const dirs = {LEFTARROW:-1, RIGHTARROW:+1};
```
Comment (paul90) [inline lib/legacy.js:54]: This should have been
``` javascript
const dirs = {[LEFTARROW]:-1, [RIGHTARROW]:+1};
```

Comment (WardCunningham) [reply lib/legacy.js:54]: Looking a few lines further down I see more complex case analysis and conclude this is really two distinct event handlers struggling to be one.

**Comment by WardCunningham** [comment]
The return was important because it was an early exit, not because anything of value was being returned. There is a structured-programming rule, single entry with single exit, that would have had us coding that as an if-then-else then no return would have been required. The early exit return should be coded as a separate statement to make it clear that the return is important for control flow.

**Pull request merged**